rets/http/parsers/parse.py

```diff
@@ -124,7 +124,8 @@ def parse_search(response: Response) -> SearchResult:
 
     return SearchResult(
         count=count,
-        max_rows=bool(elem.find('MAXROWS')),
+        # for some reason bool(elem.find('MAXROWS')) is incorrect
```

Comment:
We can remove this comment, or replace it with something like Python's `xml.etree.ElementTree.Element` objects are always considered false-y.